lib/service.js

```diff
@@ -160,21 +160,35 @@ class Service {
   serviceStatus(cb) {
     return this._execServiceStatus((err, response) => {
       if (err != null) return cb(err);
+
       let status = response.body.status;
+      let version = response.body.version;
+
+      if (version != null) {
+        version = chalk.whiteBright(`v${version}`);
+      }
+
       if (status === STATUS_CONSTANTS.ONLINE) {
-        status = chalk.green(STATUS_CONSTANTS.ONLINE);
+        status = chalk.greenBright(STATUS_CONSTANTS.ONLINE);
       }
       if (status === STATUS_CONSTANTS.UPDATING) {
-        status = chalk.yellow(STATUS_CONSTANTS.UPDATING);
+        status = chalk.yellowBright(STATUS_CONSTANTS.UPDATING);
       }
       if (status === STATUS_CONSTANTS.NEW) {
-        status = chalk.cyan(STATUS_CONSTANTS.NEW);
+        status = chalk.cyanBright(STATUS_CONSTANTS.NEW);
       }
       if (status === STATUS_CONSTANTS.ERROR) {
-        status = chalk.red(STATUS_CONSTANTS.ERROR);
```

Comment:
We should change these if statements into switch case or if-else if :grinning: Haven't noticed it so far but status is either online or updating or whatever. Anyways, we can do it any time.